### ISSUE TITLE
Accessibility - Screen reader users are unable to select date from calendar

### ISSUE THREAD

**Issue by yanbxice** [microsoft/fluentui#14029]
### Environment Information

- **Package version(s)**: @fluentui/date-time 7.10.0
- **Browser and OS versions**: Chrome + Voiceover on iOS 13.5.1

### Describe the issue:

When Voiceover is on, users are not able to navigate to the dates on the calendar view. User can only swipe from "Month Year" label to those day headers, like "SAT", "SUN". After that, the focus goes to the right box. Thus they are unable to select any date.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/yanbxice/pen/BajwqWo?editable=true%3Dhttps%3A%2F%2Ffabricweb.z5.web.core.windows.net%2Foufr%2F7.121.4%2F

#### Actual behavior:

By swiping when voiceover is on, user is unable to reach the date.

#### Expected behavior:

user should be able to navigate on the date picker and select any date they want.

**Comment by xugao** [comment]
@yanbxice - This is an issue with VoiceOver. see issue #11273. We can't do much here.

**Comment by xugao** [comment]
closing since this is external. @yanbxice - feel free to re-open if you need further assistance
